craco.config.js

```diff
@@ -0,0 +1,33 @@
+const webpack = require("webpack");
+
+module.exports = {
+  webpack: {
+    configure: {
+      resolve: {
+        fallback: {
+          // process: require.resolve("process/browser"),
+          // zlib: require.resolve("browserify-zlib"),
+          // stream: require.resolve("stream-browserify"),
+          // util: require.resolve("util"),
+          // buffer: require.resolve("buffer"),
+          // asset: require.resolve("assert"),
+          url: require.resolve("url"),
+          fs: require.resolve("fs"),
+          assert: require.resolve("assert"),
+          crypto: require.resolve("crypto-browserify"),
+          http: require.resolve("stream-http"),
+          https: require.resolve("https-browserify"),
+          os: require.resolve("os-browserify/browser"),
+          buffer: require.resolve("buffer"),
+          stream: require.resolve("stream-browserify"),
+        },
+      },
+      plugins: [
+        new webpack.ProvidePlugin({
```

Comment:
just curious, what does this plugin do?

Reply:
Not 100% sure, might be removable. 

I'm hoping this craco stuff is temporary, there's a bug in the latest CRA with webpack 5.